src/ygg_nice_ipv6.go

```diff
@@ -143,7 +148,7 @@ func main() {
 
 func matchWithinGroup(s []byte, matches int) bool {
 	m := 0
-	for b := 0; b < 15; b++ {
+	for b := 2; b < 15; b = b + 2 {
```

Comment:
First block (202, etc) and something similar to dca7:a78b, because it is not very nice